components/x-teaser-list/readme.md

```diff
@@ -0,0 +1,53 @@
+# x-teaser-list
+
+Renders a simple list of content teasers, with optional save buttons.
+
+## Installation
+
+This module is compatible with Node 6+ and is distributed on npm.
+
+```bash
+npm install --save @financial-times/x-teaser-list
+```
+
+The [`x-engine`][engine] module is used to inject your chosen runtime into the component. Please read the `x-engine` documentation first if you are consuming `x-` components for the first time in your application.
+
+[engine]: https://github.com/Financial-Times/x-dash/tree/master/packages/x-engine
+
+## Other dependencies
+
+[o-teaser](https://registry.origami.ft.com/components/o-teaser) styles will need to be imported by the consumer of this component.
+
+If selectively importing o-teaser's styles via scss, then you will need the following:
+
+```scss
+$o-teaser-is-silent: true;
+@import 'o-teaser/main';
+@include oTeaser(('default'), ('small'));
+``` 
```

Comment:
I think this section should link to the x-teaser docs as there is a risk the instructions will fall out of sync.